Snakefile

```diff
@@ -741,6 +700,40 @@ if config["monte_carlo"]["options"].get("add_to_snakefile", False) == True:
                 **config["scenario"]
             ),
 
+else:
```

Comment:
Needs to check the pre-commit locally and verify if else may be used
Currently it fails for some trailing spaces; in the past I had problems with else.

Reply:
not and issue